mws/templates/mws/show.html

```diff
@@ -206,7 +206,7 @@ <h3 class="campl-teaser-title">MWS test server settings</h3>
 
             <div class="campl-column4">
                 <div class="campl-content-container campl-side-padding">
-                    <a href="https://sliderule.mws3.csx.cam.ac.uk/dashboard/script/mws3.js?name={{ stats_name }}">
+                    <a href="{{ metrics_url }}?name={{ stats_name }}">
```

Comment:
and {% if metrics_url %} wrapper around it will make it better I think, as it is possible that metrics_url = False with the current code

Reply:
indeed, added